pom.xml

```diff
@@ -222,8 +222,28 @@
 					</dependency>
 				</dependencies>
 			</dependencyManagement>
-      </profile>
-
+        </profile>
 
+		<profile>
+			<id>Java 17</id>
+			<activation>
+				<jdk>17</jdk>
+			</activation>
+			<build>
+				<pluginManagement>
+					<plugins>
+						<plugin>
+							<groupId>org.apache.maven.plugins</groupId>
+							<artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Can't we compile and run with Java 17 without adding this?

Reply:
it is working without adding this part